autonav_ws/src/autonav_hardware/src/audible_feedback.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+
+import rclpy
+from autonav_msgs.msg import AudibleFeedback
+
+from autonav_shared.node import Node
+from autonav_shared.types import LogLevel, DeviceState, SystemState
+import time
+import os
+from just_playback import Playback
+
+# TODO: rewrite with just_playback
+
+class AudibleFeedbackConfig:
+    def __init__(self):
+        self.volume = 100
+        
+
+class AudibleFeedbackNode(Node):
+    def __init__(self):
+        super().__init__("audible_feedback_node")
+        self.write_config(AudibleFeedbackConfig())
+        self.current_playing_thread = None
+        self.tracks = []
+        self.old_system_state = self.system_state
+
+        # 
+        self.audible_feedback_subscriber = self.create_subscription(
+            AudibleFeedback, 
+            "/autonav/audible_feedback",
+            self.on_audible_feedback_received,
+            20
+        )
+
+        self.system_state_monitor = self.create_timer(0.5, self.monitor_system_state)
```

Comment:
SystemStateTransition might get implemented this year, TBD, so you might be able to change this in the future